src/Illuminate/Database/Console/TableCommand.php

```diff
@@ -52,6 +52,12 @@ public function handle(ConnectionResolverInterface $connections)
             collect($schema->listTables())->flatMap(fn (Table $table) => [$table->getName()])->toArray()
         );
 
+        if (! $schema->tablesExist([$table])) {
+            $this->warn("Table [{$table}] doesn't exist.");
```

Comment:
```suggestion
            $this->components->warn("Table [{$table}] doesn't exist.");
```

Reply:
Totally agree with components but `handle` should return `int`. If we return the warning component, it'll be `void`. What do you think?

Reply:
I've just checked the `ShowModelCommand` and it's the same there; https://github.com/laravel/framework/blob/9.x/src/Illuminate/Foundation/Console/ShowModelCommand.php#L81-L107

Reply:
Same message with PhpStorm on `ShowModelCommand`
`'void' method 'error' result used `

And for `TableCommand`: `'void' method 'warn' result used `

But it's only some warning, everything works as expected. As it's fine for you, it's fine for me :)